tests/test_oidc_client.py

```diff
@@ -0,0 +1,22 @@
+from django.test.client import Client
+from django.test.testcases import SimpleTestCase
+
+from mozilla_django_oidc.test import OIDCClient
+
+
+class TestOIDCClient(SimpleTestCase):
```

Comment:
I think deriving from `TestCase` may work better since none of the [things added by `SimpleTestCase`](https://docs.djangoproject.com/en/3.0/topics/testing/tools/#django.test.SimpleTestCase) seem needed here — are they?

Reply:
I've changed it to make use of `unittest.Testcase`. I assume that's what you mean, since the Django `TestCase` class inherits from `SimpleTestCase` :)

Reply:
It's the other way around — see https://github.com/django/django/blob/a6b3938afc0204093b5356ade2be30b461a698c5/django/test/testcases.py#L146 — but thanks for the change!

Reply:
Yes that's what I meant :) `django.TestCase` inherits from `django.SimpleTestCase` which inherits from `unittest.TestCase` :smile: Anyhoo, no worries, thanks for the feedback :)